packages/gestalt/src/Upsell.js

```diff
@@ -39,14 +42,13 @@ type UpsellActionProps = {|
 
 function UpsellAction({ data, stacked, type }: UpsellActionProps): Node {
   const color = type === 'primary' ? 'red' : 'gray';
-  const { accessibilityLabel, disabled, href, label, onClick, rel, target } = data;
+  const { accessibilityLabel, disabled, href, label, rel, target } = data;
```

Comment:
should we here not access href, rel, target, and do props.href, props.rel, props.target within the link conditional?

Reply:
DONE